pandas/core/missing.py

```diff
@@ -223,13 +223,22 @@ def _interpolate_scipy_wrapper(x, y, new_x, method, fill_value=None,
         raise ImportError('{0} interpolation requires Scipy'.format(method))
 
     new_x = np.asarray(new_x)
-
-    # ignores some kwargs that could be passed along.
-    alt_methods = {
-        'barycentric': interpolate.barycentric_interpolate,
-        'krogh': interpolate.krogh_interpolate,
-        'piecewise_polynomial': interpolate.piecewise_polynomial_interpolate,
-    }
+    import scipy
+    if scipy.__version__ <= '0.17':
```

Comment:
Actually, why don't use just start with

```python
alt_methods = {
      'barycentric': interpolate.barycentric_interpolate,
      'krogh': interpolate.krogh_interpolate,
      'from_derivatives': interpolate.BPoly.from_derivatives`,
}
```
and then only add `piecewise_polynomial` if the scipy version is less than 0.18. That way people can write code that is compatible with both versions of scipy.